### PR TITLE
#1938-Adding content view version removal stubs

### PR DESCRIPTION
**Pull request by ldjebran**
same tests stubs as for UI PR #4071 of issue #1938

### CODE REVIEW

**Comment by rplevka** [comment]
uhm, is this any different from what we already have in:
https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/api/test_contentviewversion.py#L207
?

**Comment by ldjebran** [comment]
@rplevka  many thanks, you are right  I missed to see this module.
Bring back to in progress to modify

**Comment by ldjebran** [comment]
@rplevka, changes made, bring back to review

File: tests/foreman/api/test_contentviewversion.py [hunk 1]
```diff
@@ -484,11 +484,8 @@ def test_positive_remove_cv_version_from_env_with_host_registered(self):
         5. Create an Activation key with the QE environment
         6. Register a content-host using the Activation key
         7. Remove the content view cv1 version from QE environment.
-           The remove environment wizard should propose to replace the current
-           QE environment of cv1 by an other (as QE environment of cv1
-           is attached to a content-host),
-           choose DEV and content view cv1 as a replacement for Content-host
-           and for Activation key.
+           Note - prior removing replace the current QE environment of cv1 by
```
Comment (sghai) [inline tests/foreman/api/test_contentviewversion.py]: Note - Prior to removing the QE environment, move the associated entities like CV, content-host, activation-key to other environment like 'DEV'. ?

**Comment by sghai** [comment]
ACK

**Comment by rplevka** [comment]
ACK

**Pull request merged**